src/Layer/Layer.js

```diff
@@ -108,6 +108,8 @@ class Layer extends THREE.EventDispatcher {
             config.style = new Style(config.style);
         }
         this.style = config.style || new Style();
+        this.sizeTextureTile = config.sizeTextureTile || 256;
```

Comment:
This seems more "source-dependent" than "layer-dependent" because it is related to the source data. I would rather define this in `Source`. WDYT @Desplandis ? 

Then, we should default it to `512` for `VectorTileSource`.

Also, either here or in `Source`, can you document it in the public API please?

Reply:
I agree and this seems to be what openlayers does (see https://openlayers.org/en/latest/apidoc/module-ol_source_VectorTile-VectorTile.html).